### PR TITLE
feat: use `<Helmet>` to set page title

### LINKED ISSUE #650: Use `<Helmet>` to set page title
Many places in the app set the page's title by directly changing `document.title`.  Instead `<Helmet>` should be used.

### PR DESCRIPTION
**Pull request by ckniffen**
## High Level Overview of Change

Previous code was manually setting `document.title`.  This also defines `QuickHarness` which simplifies setting up `QueryClientProvider`, `I18nextProvider`, `HelmetProvider`, and `MemoryRouter`.

Closes #650

### Context of Change

The need to define `HelmetProvided` in tests pushed me over the edge needed to create something cleaner.

### Type of Change

- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor (non-breaking change that only restructures code)
- [x] Tests (You added tests for code that already exists, or your new feature included in this PR)

### TypeScript/Hooks Update

- [x] Updated files to TypeScript

### CODE REVIEW

File: src/containers/test/utils.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import { QueryClientProvider } from 'react-query'
+import { MemoryRouter } from 'react-router'
+import type i18n from '../../i18n/testConfig'
+import { testQueryClient } from './QueryClient'
+
+export function flushPromises() {
+  return new Promise((resolve) => setImmediate(resolve))
+}
+
+export const QuickHarness: FC<
```
Comment (justinr1234) [inline src/containers/test/utils.tsx:13]: I don’t like the name. I’d go with something about app wrapper or app provider

File: src/containers/Validators/index.tsx [hunk 2]
```diff
@@ -136,6 +125,23 @@ export const Validator = () => {
       })
   }
 
+  function renderPageTitle() {
```
Comment (justinr1234) [inline src/containers/Validators/index.tsx:128]: Can you document this with jsdoc and inline comments to explain what’s going on

File: src/containers/Transactions/index.tsx [hunk 3]
```diff
@@ -159,6 +155,9 @@ export const Transaction = () => {
 
   return (
     <div className="transaction">
+      <Helmet
+        title={`${t('transaction_short')} ${identifier.substring(0, 8)}...`}
```
Comment (justinr1234) [inline src/containers/Transactions/index.tsx:159]: This 0,8 isn’t intuitive. Can you make a helper function or leave a comment

**Review by justinr1234 — APPROVED** [review]

File: src/containers/test/utils.tsx (hunk 1 above)
Comment (ckniffen) [reply src/containers/test/utils.tsx:13]: This is a testing harness.

Comment (justinr1234) [reply src/containers/test/utils.tsx:13]: TestingProvider? It’s just a nit

**Review by khancode — APPROVED** [review]
